pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -190,293 +162,146 @@ After loading the image, you can run it using the `docker run` command. Here’s
+```bash
+docker run --rm --gpus all -p 7687:7687 -p 7444:7444 memgraph/memgraph-mage:1.3-cugraph-22.02-cuda-11.5
 ```
 
-When the mgconsole starts, you can query the database: 
-
-```nocopy
-docker run -it  memgraph/mgconsole:latest
-mgconsole 1.4
-Connected to 'memgraph://127.0.0.1:7687'
-Type :help for shell usage
-Quit the shell by typing Ctrl-D(eof) or :quit
-memgraph>
-```
+## Troubleshooting
 
-You can also connect to Memgraph using the [Memgraph Lab web
-application](/data-visualization/user-manual) available within the
-`memgraph-platform` Docker image by visiting `localhost:3000`.
+### Docker not running?
 
-![](/pages/getting-started/lab.png)
+- Launch Docker Desktop and wait for it to start.
+- Run: `docker ps` to verify it's working.
+- If needed, restart your .
 
-To start using Memgraph in your application, use one of the [client
-libraries](/client-libraries) and follow their getting started guide.
+### Memgraph Lab can't connect?
 
-</Steps>
+Connection issues between **Memgraph Lab** and **Memgraph** often depend on your
+deployment method and operating system. The behavior of the
+`QUICK_CONNECT_MG_HOST` environment variable varies by platform:
 
-### Running individual tools from Memgraph Platform image
+- **Mac & Windows:**
+  Use `host.docker.internal` to allow Docker containers to communicate with
+  services running on the host machine. Set this as the value of
+  `QUICK_CONNECT_MG_HOST` when running Memgraph Lab to connect to a host-based
+  Memgraph instance:
 
-You can run only the **Memgraph database instance** from the Memgraph Platform image
-using the following command: 
+  ```bash
+  docker run -d -p 3000:3000 -e QUICK_CONNECT_MG_HOST=host.docker.internal --name lab memgraph/lab
+  ```
 
-```terminal
-docker run -p 7687:7687 memgraph/memgraph-platform -c /etc/supervisor/supervisord-memgraph-only.conf
-```
+- **Linux**:
+  You usually don't need to set `QUICK_CONNECT_MG_HOST`. It defaults to
+  `localhost`, which works if Memgraph is running directly on the host.
 
-You can run only **mgconsole** from the Memgraph Platform
-image using the following command in Linux:
+Additionally, you can specify a custom port using the `QUICK_CONNECT_MG_PORT`
+environment variable. For example:
 
-```terminal
-docker run -it --entrypoint=mgconsole memgraph/memgraph-platform
+```bash
+-e QUICK_CONNECT_MG_PORT=7688
 ```
 
-You can run only **mgconsole** application from the Memgraph Platform
-image using the following command in Windows and macOS:
+This allows Memgraph Lab to connect using a different port if needed.
 
-```terminal
-docker run -it --entrypoint=mgconsole memgraph/memgraph-platform --host host.docker.internal
-```
 
-You can also consider using a standalone `mgconsole` image as described in the [CLI section](/getting-started/cli).
+### Connection failure
 
-You can run only the **Memgraph Lab** web application from the Memgraph Platform
-image using the following command: 
+If `mgconsole` says:
```

Comment:
If mgconsole says,..
To fix this issue,..

The sentence flow is not good.

Try:
Let's say you've encountered the following error:

To fix this issue,...